docs/config/index.md

```diff
@@ -3,6 +3,157 @@ title: Config
+export default {
+  // Configuration items
+};
+```
+
+Currently dumi supports the following configuration items.
+
+## algolia
+
+- Type: `Object`
+- Defaults: `null`
+- Details:
+
+Configure Algolia's [DocSearch](https://docsearch.algolia.com/) service, usually you will need to enable the automatic generation of sitemap.xml in order to access Algolia smoothly, refer to [Configuration Item-sitemap](#sitemap).
+
+Example：
+
+```js
+{
+  algolia: {
+    apiKey: 'Your api key',
+    indexName: 'dumi',
+  }
+}
+```
+
+## description
+
+- Type: `String`
+- Defaults: `null`
+- Details:
+
+The introduction of the configuration document will be displayed under the sidebar menu title, only available in `doc` mode.
+
+## logo
+
+- Type: `String`
+- Defaults: Umi's LOGO
+- Details: 
+
+LOGO of configuration document.
+
+> If you are using a local image, such as `/public/images/xxx.png`, then configure `/images/xx.png` to import it.
+
+## locales
+
+- Type: `Array<[String, String]>`
+- Defaults: `[['en-US', 'English'], ['zh-CN', '中文']]`
+- Details:
+
+The configuration is a two-dimensional array, and the first configuration will be the default locale of the site.
+
+Each configuration is an array of length 2. The first value of the array represents the name of the locale, which will be used for splicing routing prefixes and detecting which locale the file name belongs to. The second value represents the label of the locale, which will be used Display options when switching languages.
+
+The file name suffix of the default locale is optional. For example, in the default configuration, `index.md` and `index.en-US.md` are equivalent.
+
+## mode
+
+- Type: `doc | site`
+- Defaults: `doc`
+- Details:
+
+It is used to set the presentation mode of documents. The default is document mode (left menu + right content). When configured as `site`, it can be seamlessly switched to site mode (navigation header + left menu + right content). If you want to display the text and order of the navigation menu items, you can refer to the `nav` configuration item in the frontmatter configuration.
+
+The effects of the two modes can be seen in [Guide-Multiple Presentation Modes](/zh-CN/guide/mode).
+
+## menus
+
+- Type: `Object`
+- Defaults: `Auto-generated menu`
+- Details:
+
+This configuration item is used to customize the display of the side menu. It is currently only available in the `site` mode, divided into multi-language mode and single language mode. Please refer to [Guide-Configurable Side Menu](/zh-CN/) guide/control-menu-generate#Configure side menu).
+
+## navs
+
+- Type: `Object | Array`
+- Defaults: `Auto-generated navigation`
+- Details:
+
+This configuration item is used to customize the display of the navigation bar. It is only available in the `site` mode. It can be divided into multi-language mode and single-language mode. For details, please refer to [Guide-Configuration Navigation](/zh-CN/guide/control- nav-generate#Configuration navigation).
```

Comment:
```suggestion
This configuration item is used to customize the display of the navigation bar. It is only available in the `site` mode. It can be divided into multi-language mode and single-language mode. For details, please refer to [Guide - Configuration navigation](/guide/control-nav-generate#configurate-navigation).
```